apps/rush-lib/src/cli/test/RushCommandLineParser.test.ts

```diff
@@ -217,6 +217,74 @@ describe('RushCommandLineParser', () => {
       });
     });
 
+    describe(`in repo with 'rebuild' or 'build' partially set`, () => {
+      describe(`'build' action`, () => {
+        it(`executes the package's 'build' script`, () => {
+          const repoName: string = 'overrideAndDefaultBuildActionRepo';
+          const instance: IParserTestInstance = getCommandLineParserInstance(repoName, 'build');
+          expect.assertions(8);
+          return expect(instance.parser.execute()).resolves.toEqual(true)
+            .then(() => {
+              // There should be 1 build per package
+              const packageCount: number = instance.spawnMock.mock.calls.length;
+              expect(packageCount).toEqual(2);
+
+              // Use regex for task name in case spaces were prepended or appended to spawned command
+              const expectedBuildTaskRegexp: RegExp = /fake_build_task_but_works_with_mock/;
+
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              const firstSpawn: any[] = instance.spawnMock.mock.calls[0];
+              expect(firstSpawn[SPAWN_ARG_ARGS]).toEqual(expect.arrayContaining([
+                expect.stringMatching(expectedBuildTaskRegexp)
+              ]));
+              expect(firstSpawn[SPAWN_ARG_OPTIONS]).toEqual(expect.any(Object));
+              expect(firstSpawn[SPAWN_ARG_OPTIONS].cwd).toEqual(path.resolve(__dirname, `${repoName}/a`));
+
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              const secondSpawn: any[] = instance.spawnMock.mock.calls[1];
+              expect(secondSpawn[SPAWN_ARG_ARGS]).toEqual(expect.arrayContaining([
+                expect.stringMatching(expectedBuildTaskRegexp)
+              ]));
+              expect(secondSpawn[SPAWN_ARG_OPTIONS]).toEqual(expect.any(Object));
+              expect(secondSpawn[SPAWN_ARG_OPTIONS].cwd).toEqual(path.resolve(__dirname, `${repoName}/b`));
+            });
+        });
+      });
+
+      describe(`'rebuild' action`, () => {
+        it(`executes the package's 'rebuild' (not 'build') script`, () => {
```

Comment:
Do we want this? I think it should still call the `build` script, just not incrementally.